src/renderer/components/tree-view/tree-label-edit.js

```diff
@@ -19,20 +19,25 @@ export default class LabelEditor extends Component {
   };
 
   state = {
-    title: 'Untitled'
+    title: 'Untitle'
   };
 
-  handleBlur() {
-    this.props.onDismiss();
+  _handleChange(e) {
```

Comment:
Is there a reason why all the handlers now have `_` at the beginning? I know it makes private methods obvious but let's keep the repo's convention for now, what do you think?

Reply:
I think they help to code readability. I am okay with naming without underline, also repo's convention is important anyway. I remove those underlines.